### DISCUSSION TITLE
É possível importar uma tabela legada para o prisma?

### DISCUSSION THREAD

**Discussion by eliasfr** [prisma/prisma#11610] [Q&A] [answered]
I would like to create the models in the prism, but without generating them as migrations and without changing the existing database and tables in mysql.

**Comment by pantharshit00** [comment]
Hello!

Can you please post this again in English? We can't answer or moderate any content that is not in English.

**Comment by janpio** [comment, marked as answer]
> Is it possible to import a legacy table into prisma?

Yes, you can just use Prisma Introspection: https://www.prisma.io/docs/concepts/components/introspection with the command `prisma db pull`. This will create a schema of your current database and tables that you can use in Prisma.

**Comment by eliasfr** [comment]
Thank you for your help

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
